timeseries/tests/unittests/models/test_local.py

```diff
@@ -86,25 +86,6 @@ def test_when_local_model_saved_then_local_model_args_are_saved(model_class, hyp
     assert dict_equal_primitive(model._local_model_args, loaded_model._local_model_args)
 
 
-@pytest.mark.parametrize("model_class", TESTABLE_MODELS)
```

Comment:
We already have the same test here https://github.com/autogluon/autogluon/blob/master/timeseries/tests/unittests/models/test_models.py#L269.